ThingIFSDK/ThingIFSDK/CommandForm.swift

```diff
@@ -53,33 +45,27 @@ open class CommandForm: NSObject, NSCoding {
     /**
     Initializer of CommandForm instance.
 
-    - Parameter schemaName: Schema name.
-    - Parameter schemaVersion: Schema version.
-    - Parameter actions: List of actions. Must not be empty.
+    - Parameter actions: List of actions. Must not be empty. Both of
```

Comment:
May be `Array of actions` is better than `List of actions`. There is not `List` in swift.